RevenueCatUI/CustomerCenter/Views/FeedbackSurveyView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  FeedbackSurveyView.swift
+//
+//
+//  Created by Cesar de la Vega on 12/6/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@available(visionOS, unavailable)
+struct FeedbackSurveyView: View {
+
+    @ObservedObject
+    var feedbackSurveyData: FeedbackSurveyData
+
+    var body: some View {
+        VStack {
+            Text(feedbackSurveyData.configuration.title)
+                .font(.title)
+                .padding()
+
+            Spacer()
+
+            FeedbackSurveyButtonsView(options: feedbackSurveyData.configuration.options,
+                                      action: feedbackSurveyData.action)
+        }
+    }
+
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@available(visionOS, unavailable)
+struct FeedbackSurveyButtonsView: View {
+
+    let options: [CustomerCenterConfigData.HelpPath.FeedbackSurvey.Option]
+    let action: (() -> Void)
+
+    var body: some View {
+        VStack(spacing: 16) {
+            ForEach(options, id: \.id) { option in
+                Button(option.title) {
+                    Task {
+                        self.action()
```

Comment:
Hmm so the way it is right now, every option will have the same action. However, we might have different actions right? Like some questions might provide a promotional offer and some won't. Do we need to change this and/or send back the id?

Reply:
You're correct. I built that in https://github.com/RevenueCat/purchases-ios/pull/3968/files#diff-b11601f94f84f34e8c7259de27e1760a2919c2ccb27d582eb7d4daa019d57f37 to keep this one simple, I should have make that more clear sorry!